README.md

```diff
@@ -64,20 +67,26 @@ yarn add contentful-management
 If you'd like to use a standalone built file you can use the following script tag or download it from [unpkg](https://unpkg.com), under the `dist` directory:
 
 ``` html
-<script src="https://unpkg.com/contentful-management@latest/dist/contentful-management.min.js"></script>
+<script src="https://unpkg.com/contentful-management@latest/dist/contentful-management.browser.min.js"></script>
```

Comment:
To make it clear that this is a browser target. It won't work on node thanks to weird axios bundeling behavior. Since it is a major version, renaming is not a problem.